code/workspace/linting-and-testing/example_sqs_queue.yaml

```diff
@@ -0,0 +1,25 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - linting and testing - example SQS queue (uksb-1q9p31idr)
+
+Resources:
+  MyQueue:
```

Comment:
Please rename `MyQueue` to something more descriptive e.g. `CfnWorkshopQueue` or just plain simple `SqsQueue`

Reply:
Thank you, will use `SqsQueue`.